### PR TITLE
Refactor/refactor LearningCourse container

### PR DESCRIPTION
**Pull request by ostapdans99**
In this PR i just reorgonized and make more understandable logic in LearningContainer. I didn't change anything from CSS. I just moved everything to different folders. All CSS changes will occur within my other task V1-173.

### CODE REVIEW

File: frontend/src/pages/LearningCourse/ActionButton/styled.ts [hunk 1]
```diff
@@ -0,0 +1,18 @@
+import styled from 'styled-components';
+
+import theme from 'themeSettings';
+
+export const ActionButtonWrapper = styled('div')({
+  display: 'inline-block',
+  verticalAlign: 'top',
+  width: '155px',
+  marginLeft: '40px',
+  color: 'white',
```
Comment (irina-sinitsyna) [inline frontend/src/pages/LearningCourse/ActionButton/styled.ts:10]: Please use hex or theme.palette instead

File: frontend/src/pages/LearningCourse/CourseInfoToggle/CourseInfoToggle.tsx [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+
+import { CourseInfoToggleButton, ExpandLessIcon, ExpandMoreIcon } from './styled';
+
+interface IProps {
+  isCourseInfoOpen: boolean;
+  toggleCourseInfoOpen: () => void;
+}
+
+const CourseInfoToggle: FC<IProps> = ({ isCourseInfoOpen, toggleCourseInfoOpen }) => {
+  return (
```
Comment (irina-sinitsyna) [inline frontend/src/pages/LearningCourse/CourseInfoToggle/CourseInfoToggle.tsx:11]: please remove return statement

File: frontend/src/pages/LearningCourse/CourseInfoToggle/styled.ts [hunk 3]
```diff
@@ -0,0 +1,30 @@
+import styled from 'styled-components';
+
```
Comment (irina-sinitsyna) [inline frontend/src/pages/LearningCourse/CourseInfoToggle/styled.ts:2]: extra empty line

File: frontend/src/pages/LearningCourse/CourseInfoToggle/styled.ts [hunk 4]
```diff
@@ -0,0 +1,30 @@
...
+import { ExpandMore, ExpandLess } from '@mui/icons-material';
+
+import theme from 'themeSettings';
+
+export const CourseInfoToggleButton = styled('div')({
+  display: 'inline-block',
+  marginRight: 'calc(100% - 294px) !important',
+  height: '40px',
+  padding: '8px 0px 12px 0px',
+  fontFamily: '"Ubuntu", sans-serif',
```
Comment (irina-sinitsyna) [inline frontend/src/pages/LearningCourse/CourseInfoToggle/styled.ts:12]: maybe you can declare fontFamily only for parent component, not for each component?

File: frontend/src/pages/LearningCourse/LearningCourse.tsx [hunk 5]
```diff
@@ -1,73 +1,49 @@
-import React from 'react';
+import { FC } from 'react';
 import { Link } from 'react-router-dom';
 
 import { AuthorizedLayout } from 'components/Layout';
-import { Back, Forward } from 'components/Arrows';
-import { MATERIAL } from 'constants/materials';
-import { ClientCourse } from 'types/clientCourse';
-import { CustomButton } from 'components/Button/styled';
+
```
Comment (irina-sinitsyna) [inline frontend/src/pages/LearningCourse/LearningCourse.tsx:5]: fix imports order please

File: frontend/src/pages/LearningCourse/CourseInfoToggle/styled.ts [hunk 6]
```diff
@@ -0,0 +1,30 @@
...
+  },
+});
+
+export const ExpandMoreIcon = styled(ExpandMore)({
+  color: '#131313',
+  verticalAlign: 'middle',
+});
+
+export const ExpandLessIcon = styled(ExpandLess)({
+  color: '#131313',
```
Comment (irina-sinitsyna) [inline frontend/src/pages/LearningCourse/CourseInfoToggle/styled.ts:28]: could you please add this color to theme.palette (cause it's a commonly used color) and reuse it here?

**Review by irina-sinitsyna — CHANGES_REQUESTED** [review]

File: frontend/src/pages/LearningCourse/CourseInfoToggle/styled.ts (hunk 3 above)
Comment (ostapdans99) [reply frontend/src/pages/LearningCourse/CourseInfoToggle/styled.ts:2]: done

File: frontend/src/pages/LearningCourse/ActionButton/styled.ts (hunk 1 above)
Comment (ostapdans99) [reply frontend/src/pages/LearningCourse/ActionButton/styled.ts:10]: done

File: frontend/src/pages/LearningCourse/CourseInfoToggle/styled.ts (hunk 4 above)
Comment (ostapdans99) [reply frontend/src/pages/LearningCourse/CourseInfoToggle/styled.ts:12]: done

File: frontend/src/pages/LearningCourse/CourseInfoToggle/styled.ts (hunk 6 above)
Comment (ostapdans99) [reply frontend/src/pages/LearningCourse/CourseInfoToggle/styled.ts:26]: I will change it occur within my other task V1-173

**Pull request merged**